include/deal.II/base/mpi.h

```diff
@@ -1609,6 +1793,42 @@ namespace Utilities
 
 
 
+    template <typename T>
+    void
+    broadcast(T *                buffer,
+              const size_t       count,
```

Comment:
Since you're using a signed type for `count`, can you add an
```
  Assert (count >= 0, ExcMessage ("The number of objects you want to send cannot be negative."));
```

Reply:
Good point. Added a second Assert for good measure.